tools/driver/driver.cpp

```diff
@@ -180,7 +180,7 @@ static int run_driver(StringRef ExecName,
 }
 
 int main(int argc_, const char **argv_) {
-  SmallVector<const char *, 256> argv(&argv_[0], &argv_[argc_]);
+  SmallVector<const char *, 256> args(&argv_[0], &argv_[argc_]);
```

Comment:
Can you give this a more descriptive name, then? It's not good to have `args` and `argv` both floating around in this function.